features/facebookImport/src/model/entities/related-accounts-group.js

```diff
@@ -1,5 +1,8 @@
-export default class RelatedAccountsGroup {
+import EntitiesGroup from "./entities-group";
```

Comment:
This again looks like a hierarchy of classes that should be possibly moved to a lower level.